backend/dummy.go

```diff
@@ -36,167 +35,200 @@ type OrderInfo struct {
 	// SEARCH BAR ON THIS PAGE https://www.amazon.com/gp/css/returns/homepage.html?ref_=footer_hy_f_4
 }
 
-var (
-	// ErrNameNotProvided is thrown when a name is not provided
-	ErrNameNotProvided = errors.New("no name was provided in the HTTP body")
+const (
+	NAME_COLUMN           = 2
+	DATE_ORDERED_COLUMN   = 0
+	ASIN_COLUMN           = 4
+	ORIGINAL_PRICE_COLUMN = 12
 )
 
 // Handler is Lambda function handler
 func Handler(request string) (ItemCategories, error) {
-	// If no name is provided in the HTTP request body, throw an error
+	//note: we must return a valid ItemCategories struct, it cannot be nil
+	// emptyRequest is thrown when the body of the Lambda Request is empty
 	if len(request) < 1 {
-		return ItemCategories{}, errors.New("This was a bad request: " + request)
+		return ItemCategories{}, errors.New("Handler: Bad Request - Empty body")
 	}
-
-	return New(request)
+	return newItemCategories(request)

```

Comment:
i think a nicer model is to do 
`orderPriceChanges, err :=  getOrderPriceChanges(ordersString)
if err != nil {
    // log error
    return OrderPriceChanges{}, nil
}

return orderPriceChanges, nil`

and in that same vein, i think variable names can be changed around to what i have used :)

feel free to suggest better names if you think of any